assets/javascripts/modules/youtubePlayer.js

```diff
@@ -0,0 +1,83 @@
+require('jquery');
+var GOVUK = require('stageprompt');
+
+/**
+ * Embed a Youtube iframe in the page, replacing a div with the js-youtube-player class.
+ * Attributes required for player parameters:
+ *
+ *  id: required to render the iframe, value does not make a difference.
+ *  data-video-id: the id of the Youtube video to play.
+ *  data-ga-play-event: Google Analytics event to fire when video is played, should be in category:event:label format. omit if no GA event required.
+ *
+ * Usage example:
+ *
+ *   <div class='youtube-player-container'>
+ *      <div id='youtube-player' class='js-youtube-player' data-video-id='MlMnz0Omosk' data-ga-play-event='tax-return-help:Click:play'></div>
+ *   </div>
+ */
+
+module.exports = function() {
+
+  var YoutubePlayer = YoutubePlayer || {};
```

Comment:
would YoutubePlayer ever exist here? this could be `var YoutubePlayer = {};` right?